src/drt/README.md

```diff
@@ -17,15 +17,173 @@ guide format.
+| `-shared_volume` | The mount path of the nfs shared folder. |
+| `-cloud_size` | The number of workers. |
+| `-clean_patches` | Clean unneeded patches during detailed routing. | 
+| `-no_pin_access` | Disables pin access for routing. |
+| `-min_access_points` | Minimum access points for standard cell and macro cell pins. | 
+| `-save_guide_updates` | Flag to save guides updates. |
+| `-repair_pdn_vias` | This option is used for some PDKs which have M1 and M2 layers that run in parallel. |
+
+### Detailed Route debugging
+
+```tcl
+detailed_route_debug 
+    [-pa]
+    [-ta]
+    [-dr]
+    [-maze]
+    [-net name]
+    [-pin name]
+    [-worker x y]
+    [-iter iter]
+    [-pa_markers]
+    [-dump_dr]
+    [-dump_dir dir]
+    [-pa_edge]
+    [-pa_commit]
+    [-write_net_tracks]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-pa` | Enable debug for pin access. |
+| `-ta` | Enable debug for track assignment. |
+| `-dr` | Enable debug for detailed routing. |
+| `-maze` | Enable debug for maze routing. | 
+| `-net` | Enable debug for net name. |
+| `-pin` | Enable debug for pin name. |
+| `-worker` | Debugs routes that pass through the point `{x, y}`. |
+| `-iter` | Specifies the number of debug iterations. (default 0, integer) |
+| `-pa_markers` | Enable pin access markers. |
+| `-dump_dr` | Filename for detailed routing dump. |
+| `-dump_dir` | Directory for detailed routing dump. |
+| `-pa_edge` | Enable visibility of pin access edges. |
+| `-pa_commit` | Enable visibility of pin access commits. |
+| `-write_net_tracks` | Enable writing of net track assigments. |
+
+### Check Pin access 
+
+```tcl
+pin_access
+    [-db_process_node name]
+    [-bottom_routing_layer layer]
+    [-top_routing_layer layer]
+    [-min_access_points count]
+    [-verbose level]
+    [-distributed]
+    [-remote_host rhost]
+    [-remote_port rport]
+    [-shared_volume vol]
+    [-cloud_size sz]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-db_process_node` | Specify process node. |
+| `-bottom_routing_layer` | Bottommost routing layer. |
+| `-top_routing_layer` | Topmost routing layer. |
+| `-min_access_points` | Minimum number of access points per pin. |
+| `-verbose` | Sets verbose mode if the value is greater than 1, else non-verbose mode (must be integer, or error will be triggered.) |
+| `-distributed` | Enable distributed pin access algorithm. |
+| `-remote_host` | The host IP. |
+| `-remote_port` | The value of the port to access from. |
+| `-shared_volume` | The mount path of the nfs shared folder. |
+| `-cloud_size` | The number of workers. |
+
+### Useful developer functions
+
+If you are a developer, you might find these useful. More details can be found in the [source file](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/drt/src/TritonRoute.cpp) or the [swig file](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/drt/src/TritonRoute.i).
```

Comment:
Has the need for a full link issue been resolved? I think I saw relative links in other PRs. Please use the same format for all.